CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+# Contributing
+
+For general contributing guidelines, see the [Contributing Guide on DoneJS.com][].
```

Comment:
Good points, I can inline the links. Did you want me to do that for the repeated local links to `package.json` too? I kind of like them being reference links (the two local `package.json` links). I often read markdown files raw in vim, and this is an improvement when reading documentation that way.